openstack/standalone-cinder/pkg/provisioner/provisioner.go

```diff
@@ -19,13 +19,17 @@ package provisioner
 import (
 	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/golang/glog"
 	"github.com/gophercloud/gophercloud"
 	"github.com/kubernetes-incubator/external-storage/lib/controller"
 	"github.com/kubernetes-incubator/external-storage/openstack/standalone-cinder/pkg/volumeservice"
 	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/client-go/kubernetes"
+
+	volumes_v2 "github.com/gophercloud/gophercloud/openstack/blockstorage/v2/volumes"
```

Comment:
Maybe better left for another patch, but the Cinder v2 API is deprecated and all clients should be using v3. New deployments (as of the Pike release) do not have the v2 API enabled by default.

Reply:
Thanks for reminding me of this!  I'd like to adopt v3 but this will be a larger effort (especially due to some changing auth semantics.